Composer/packages/types/src/orchestrator.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type IOrchestratorModelRequest = {
+  type: 'en_intent' | 'multilingual_intent';
```

Comment:
nit: type is a reserved keyword, I suggest using ```somethingType``` or ```kind```

Reply:
Thanks @hatpick , I was using the convention that others used here. Is `interface` ok?

Reply:
with the current type, change ```IOrchestratorModelRequest``` to ```OrchestratorModelRequest``` (removing the prefix I)

Reply:
I meant the key you use here, ```type```, in strict mode it will complain about it but I guess we are not in that mode. It's still a good practice if you replace it with somethingType or kind:
```type: 'en_intent' | 'multilingual_intent';```

Reply:
Ah ok understood thanks, will change to `kind`.

Reply:
Changed to `kind` here, enclosed the `orchestrator` controller in a class, added all other requested changes.  Thanks @hatpick